docker/kot/Dockerfile

```diff
@@ -13,14 +13,9 @@ COPY --from=builder /usr/share/zoneinfo/Asia/Tokyo /etc/localtime
 ENV PYTHONPATH=/usr/local:$PYTHONPATH
 ENV APP_HOME /kot
 WORKDIR $APP_HOME
-RUN apt -y update \
-  && apt -y install chromium \
-  && rm -rf /var/lib/apt/lists/*
```

Comment:
kot supports Lambda execution, which is achieved by deploying container images. So, container image with chromium is required and you cannot remove these lines.
How about creating another Dockerfile for compose file?

Reply:
I misunderstood, Dockerfile for lambda exists. So, removing these line is no problem.